.dockerignore

```diff
@@ -0,0 +1,8 @@
+# ignore everything
+*
+
+# Except the following
+!micro_g
+!micro_g_description
```

Comment:
Forgot to delete this

Reply:
I'm replying to [this](https://github.com/dawsonc/micro-g-robotics/pull/2#discussion_r1349830424) here because it wouldn't let me reply in a thread. I will delete it: I added the comment as a first pass in the review and will commit the change once the initial review is done so that it's all in a single commit